hooks/src/internal.d.ts

```diff
@@ -1,11 +1,16 @@
 import {
 	Component as PreactComponent,
-	PreactContext
+	PreactContext,
+	Internal as PreactInternal
 } from '../../src/internal';
 import { Reducer } from '.';
 
 export { PreactContext };
 
+export type Internal = PreactInternal & {
+	_component: Component;
```

Comment:
Doesn't the `PreactInternal` interface already have this field? Was TS complaining about the `| null` on the core Internal type?

Reply:
mainly because we extend `Component` to have `__hooks`